website/views/options/social/group.jade

```diff
@@ -22,7 +22,7 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
                 span.glyphicon.glyphicon-ban-circle
                 =env.t('leave')
               a.btn.btn-success.pull-right(ng-if='!isMemberOfGroup(User.user._id, group)', ng-click='join(group)')=env.t('join')
-            span(ng-if='group.leader._id == user.id')
+            span(ng-if='group.leader._id == user.id || (user.contributor.admin === true && group.type === "guild")')
```

Comment:
@crookedneighbor 
Good point, thanks. It looks like you specifically highlighted just the administrator portion of this condition, so I'll plan on only moving that logic to the controller. But could we also abstract the whole thing into a method like `$scope.isAbleToEditGroup`?